slapo/op/fused_bias.py

```diff
@@ -118,3 +109,52 @@ def reset_parameters(self, prev_weight=None):
 
     def forward(self, inp):
         return self.func(inp, self.bias)
+
+
+def bias_dropout(
+    x: torch.Tensor,
+    bias: torch.Tensor,
+    p: float = 0.5,
+    training: bool = True,
+    inplace: bool = False,
+) -> torch.Tensor:
+    return F.dropout(x + bias, p=p, training=training, inplace=inplace)
```

Comment:
when inplace can be true?

Reply:
I don't know actually. This is to align the signature of F.dropout.